libraries/core/src/test/java/com/paritytrading/philadelphia/FIXInitiatorSocketTest.java

```diff
@@ -0,0 +1,26 @@
+package com.paritytrading.philadelphia;
```

Comment:
Could you please add the license boilerplate to this file (with a 2022 copyright)?